### ISSUE TITLE
Awesome work!

### ISSUE THREAD

**Issue by Oaluiser** [YoanDimitrov97/OPTCG_CardScraper#1]
The English version of the official site doesn't have the card list yet, but would be great to do the same when they release it.

Also, I might say that you're downloading only the BOOSTER PACK -ROMANCE DAWN- [OP-01], but in the search tab there are also other 6 pages for starter decks and promotion cards.

**Comment by YoanDimitrov97** [comment]
Yes, once official english translations drop I will change it to scrape all of them.
I didn't notice there were more cards that I haven't scrapped, I thought that page url showed every card in the game so far, but I will definitely look to scrape all of them once I have the chance.

**Comment by YoanDimitrov97** [comment]
Now it downloads all the cards, just waiting on official images in english.
Thank you for noticing the issue.
